.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
```

Comment:
This is a public repo and we usually use template like https://github.com/snyk/nuget-semver/pull/7/files in those.

Reply:
Will chat internally to see what we want to do for this one as it is a fork